packages/ra-core/src/auth/useAuthenticated.spec.tsx

```diff
@@ -146,9 +146,14 @@ describe('useAuthenticated', () => {
             expect(authProvider.checkAuth).toHaveBeenCalledTimes(1);
         });
         expect(authProvider.checkAuth.mock.calls[0][0]).toEqual({});
-        await waitFor(() => {
-            expect(authProvider.logout).toHaveBeenCalledTimes(1);
-        });
+        await waitFor(
+            () => {
+                expect(authProvider.logout).toHaveBeenCalledTimes(1);
+            },
+            {
+                timeout: 4000,
+            }
```

Comment:
please compactify on 1 line

Reply:
prettier. I could add an `prettier-ignore` but that will be worse IMO